lib/hammer_cli_foreman/combinations.rb

```diff
@@ -0,0 +1,72 @@
+module HammerCLIForeman
+  class Combinations < HammerCLIForeman::Command
+    resource :template_combinations
+    command_name 'combinations'
+    desc _("Manage template combinations")
+
+
+    module RequestParams
+      def request_params
+        params = super
+        params["template_combination"] =  {}
+        params["template_combination"]["hostgroup_id"] = params["hostgroup_id"].to_s
+        params["template_combination"]["environment_id"] = params["environment_id"].to_s
+
+        params
+      end
+    end
+
+    class InfoCombination < HammerCLIForeman::InfoCommand
+      output do
+        field :id, _('Id')
+        field :provisioning_template_id, _('provisioning_template_id')
```

Comment:
The descriptions should be human readable and properly capitalized so `_('Provisioning template ID')`